app/main.py

```diff
@@ -16,4 +16,8 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable" : [my_favourite_films, marks, collection_of_coins],
+    "immutable" : [lucky_number, pi, one_is_a_prime_number, name, profile_info]
```

Comment:
The `collection_of_coins` variable is a set, which is an immutable collection. It should be placed in the 'immutable' list instead of the 'mutable' list.